### PR TITLE
winapi: Implement GetSystemTime()

### PR DESCRIPTION
**Pull request by sam-itt**
Implement GetSystemTime using KeQuerySystemTime.

### CODE REVIEW

File: lib/winapi/minwinbase.h [hunk 1]
```diff
@@ -17,4 +17,15 @@ typedef struct _OVERLAPPED
...
 
+typedef struct _SYSTEMTIME {
+  WORD wYear;
+  WORD wMonth;
+  WORD wDayOfWeek;
+  WORD wDay;
+  WORD wHour;
+  WORD wMinute;
+  WORD wSecond;
+  WORD wMilliseconds;
```
Comment (JayFoxRox) [inline lib/winapi/minwinbase.h]: Indentation differs from nearby code like `_OVERLAPPED`

File: lib/winapi/sysinfo.c [hunk 2]
```diff
@@ -2,6 +2,28 @@
 #include <assert.h>
 #include <xboxkrnl/xboxkrnl.h>
 
+void GetSystemTime (LPSYSTEMTIME lpSystemTime)
+{
+    assert(lpSystemTime != NULL);
+
+    LARGE_INTEGER KTime;
+    TIME_FIELDS TimeFields;
```
Comment (JayFoxRox) [inline lib/winapi/sysinfo.c]: nit: Other code nearby appears to use `thatStyle` instead.

File: lib/winapi/sysinfo.c [hunk 3]
```diff
@@ -2,6 +2,28 @@
...
+
+    KeQuerySystemTime(&KTime);
+
```
Comment (JayFoxRox) [inline lib/winapi/sysinfo.c]: nit: I feel like this blank line doesn't serve any purpose.

File: lib/winapi/minwinbase.h (hunk 1 above)
Comment (sam-itt) [reply lib/winapi/minwinbase.h]: Done

File: lib/winapi/sysinfo.c (hunk 2 above)
Comment (sam-itt) [reply lib/winapi/sysinfo.c]: Done

File: lib/winapi/sysinfo.c (hunk 3 above)
Comment (sam-itt) [reply lib/winapi/sysinfo.c]: Removed blank line 13 to make a bloc after declarations

**Comment by thrimbor** [comment]
Works fine, thanks for submitting. I've taken the liberty to clean up a stray whitespace. Merging.

**Pull request merged**